bool obj2pose_request
---
bool obj2pose_respond